src/frontends/paddle/src/op/roi_align.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs roi_align(const NodeContext& node) {
+    const auto data_node = node.get_input("X");
+    const auto roi_node = node.get_input("ROIs");
+    // TODO: support 'aligned' feature #82319
+    const auto aligned = node.get_attribute("aligned", false);
+    PADDLE_OP_CHECK(node, !aligned, "OpenVINO not support 'aligned' feature!");
+
+    // limitation for batch_size = 1
+    if (data_node.get_partial_shape().rank().is_static() && data_node.get_partial_shape()[0].is_static())
+        PADDLE_OP_CHECK(node, data_node.get_partial_shape()[0] == 1, "roi_align currenty only support batch_size = 1!");
```

Comment:
about "any problem to enable batch_size>1" please see the previous comments of this PR.
"is it a separate PR" yes, because it's not blocked our current model target. so it should be a separate PR.

Reply:
Could you please put ticket ID in comments to track batch_Size>1 enabling?